src/shapes/point.py

```diff
@@ -0,0 +1,36 @@
+from numpy import ndarray
+
+from src.shapes.shape import Shape
+import cv2
+
+
+class Point(Shape):
+
+    def __init__(self, x: float, y: float, line_color: tuple[float, float, float], image: ndarray):
+        self.x = x
+        self.y = y
+        self.line_color = line_color
```

Comment:
since point in your case is more of an abstract shape in essence in your code (other cases go to circle), it should not get line_color and be built from the line color of it's containing shape